src/commands/lambda/flare.ts

```diff
@@ -271,6 +286,44 @@ export class LambdaFlareCommand extends Command {
   }
 }
 
+/**
+ * Validate the start and end flags and adds error messages if found
+ * @param start start time as a string
+ * @param end end time as a string
+ * @throws error if start or end are not valid numbers
+ * @returns [startMillis, endMillis] as numbers
+ */
+export const validateStartEndFlags = (start: string | undefined, end: string | undefined) => {
```

Comment:
Do we cover edge caes for negative numbers?

Reply:
The command doesn't run when trying to enter a negative number for start/end flags:

<img width="1419" alt="Screenshot 2023-07-11 at 7 58 52 PM" src="https://github.com/DataDog/datadog-ci/assets/29668820/163f7aa5-89ba-4432-8d75-0975d69a7b2b">